README.md

```diff
@@ -5,14 +5,16 @@ A curated list of awesome projects built with [Hyperapp](https://github.com/hype
 ## Contents
 - [Apps and Boilerplates](#apps-and-boilerplates)
 - [Libraries and Mixins](#libraries-and-mixins)
-- [Resources and Others](#resources-and-others)
+- [Official Resources and Community](#official-resources-and-community)
+- [Other Resources](#other-resources)
 - [Contributing](#contributing)
 
 ## Apps and Boilerplates
 
 - [deployable](https://github.com/lukejacksonn/deployable) - A ready-to-deploy static web app template built with Hyperapp.
 - [happy](https://github.com/zaceno/happy) - Happiness index calculator for teams, built with Hyperapp.
 - [hyperapp-boilerplate](https://github.com/tzellman/hyperapp-boilerplate) - Boilerplate showing how to create a production-ready Hyperapp application.
+- [hyperapp-electron](https://github.com/lukejacksonn/hyperapp-electron) - A minimal electron app starter with hyperapp, livereload and redux devtools
```

Comment:
- ~electron~ Electron
- ~hyperapp~ Hyperapp
- ~livereload~ LiveReload
- ~redux~ Redux
- ~devtools~ DevTools
- add a full stop to the end of the line

Reply:
Thanks for the careful check. I made the requested changes. I am doubtful that LiveReload should be a proper noun in this case but no big deal, I changed it.